ern-runner-gen-android/src/hull/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.walmartlabs.ern" >
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.walmartlabs.ern.{{{lowerCaseMiniAppName}}}">
```

Comment:
Minor: Can you keep the line break after `"http://schemas.android.com/apk/res/android"`? No need to modify line 2 in this file.. ;)